docker-compose.yml

```diff
@@ -39,13 +39,13 @@ services:
     environment:
       MONGO_INITDB_ROOT_USERNAME: ${MONGODB_USERNAME}
       MONGO_INITDB_ROOT_PASSWORD: ${MONGODB_PWD}
+      MONGO_INITDB_DATABASE: ${MONGODB_DB_NAME}
     ports:
       - "27017:27017"
     volumes:
-      - mongodb_data:/data/db
```

Comment:
The mongodb_data volume was created to maintain the state of the database when you start/stop the container. You can view the volume in Docker Desktop: 
![image](https://user-images.githubusercontent.com/44559579/207393762-817a6532-1699-4f25-891d-2446a8c742e8.png)

Do you know if the new volume you created also stores the data from the database? You can test this by populating your DB, and restarting the container to see if the records still exist. If not, can we keep both lines?

Reply:
I believe that it will persist even if you restart the database. I just added a random document under the users collection and then restarted, and the record was still there.

Here's a screen recording of what I saw -

https://user-images.githubusercontent.com/69999501/207406893-a08c5f99-4947-4eec-8361-8d354c22629c.mp4

Reply:
Cool! Just to be sure, can you open the volumes tab on the left-hand side of Docker Desktop and check if you see something like a `mongo-init.js` entry?

Reply:
I can do some testing with this later by building my environment from scratch, but thinking about it now I think we may need to keep both since they each serve a different purpose. One maps the volume to the `data/db` directory of the Mongo container  (i.e. persisting data) and the other is for linking scripts to the `docker-entrypoint-initdb.d` directory which are ran at start-up.